documentation/src/main/asciidoc/topics/con_query_non_indexed.adoc

```diff
@@ -0,0 +1,10 @@
+[id='non-indexed-queries_{context}']
+= Non-indexed queries
+
+{brandname} recommends indexing caches for the best performance for queries.
+However you can query caches that are non-indexed but you must configure them to use ProtoStream encoding, as follows:
```

Comment:
This is only valid for caches in infinispan server. For embedded mode, you can query caches without indexes using POJOs and no encoding is necessary (it assumes application/x-java-object)